### DISCUSSION TITLE
[Technical Questions] Package Resolution Failed for SpeziFhir

### DISCUSSION THREAD

**Discussion by kcallon** [CS342/.github#58] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Swift Package Manager

### Description

When I try to add spezifhir, I have a package resolution failed issue. As pictured below.
<img width="1218" alt="Screenshot 2024-02-03 at 7 31 44 PM" src="https://github.com/CS342/.github/assets/70660419/13ae868f-08fc-4dcb-a0b4-a14071bd7990">

### Reproduction

To add spezifhir, I tried "add packages" and copied the github link into the search bar. 

### Expected behavior

I was expecting the package to be added. 

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by philippzagar** [comment]
Hi @kcallon, thanks for creating the Issue! 🚀 
As we discussed in last week's class, this seems like a dependency resolution error as your application depends on specific versions of packages that differ from the ones of the dependencies themselves.
You've already created another office hour issue: Let's discuss the dependency resolution within one of the office hours 👍 
Otherwise, feel free to share the name of the feature branch with me, then I have more context and can play around with it!

**Reply by PSchmiedmayer** [reply]
@kcallon I hope this issue has already been resolved. If so, can you and @philippzagar share what the solution was in the office hours so anyone who stumbles across the same issue can follow your solution ideas?

**Reply by kcallon** [reply, marked as answer]
Yes! So I didn't realize when I pulled a teammates code it was important to restart my xcode for the package.resolved file to update properly.

**Reply by PSchmiedmayer** [reply]
@kcallon Thank you for sharing your solution; great!
